doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -173,21 +173,97 @@ context broker, context registry, context consumer)
+  * `value`: for the attribute value, rendered as a regular JSON value (string, number, boolean or null), object or array.
+    This property is optional in requests, meaning that the attribute has `null` value. It is always present in responses.
+  * `type`: a JSON string which represents the user-defined NGSI attribute type. This property is optional in requests
+    meaning that the attribute doesn't have a NGSI type. In responses, this property is set to `null` if the attribute
+    doesn't have a type.
+  * `metadata`: a JSON object which represents attribute metadata. This property is optional in requests, meaning that the
+    attribute doesn't have metadata. In responses, this property is set to `{}` if the attribute doesn't have metadata.
+    Each property in this object represents a metadata element (which name is the name of the property) which value is
+    a JSON object composed of two properties: `value` and `type` (`value` and `type` following the same rules that the
+    ones described in the previous two bullets).
+
+## JSON attribute representation
+
+Attributes are represented in request and response payloads by a JSON object following this pattern:
+
+```
+{
+  "value": <...>,
+  "type": <...>,
+  "metadata": <...>
+}
+```
+
+where:
+
+* `value`: represents the attribute value, rendered as a regular JSON value (string, number, boolean or null), object or array.
+  This property is optional in requests, meaning that the attribute has `null` value. It is always present in responses.
+* `type`: a JSON string which represents the user-defined NGSI attribute type. This property optional in requests
+  meaning that the attribute doesn't have a NGSI type. In responses, this property is set to `null` if the attribute doesn't have
+  a type.
+* `metadata`: a JSON object which represents attribute metadata. This property is optional in requests, meaning that the
+  attribute doesn't have metadata. In responses, this property is set to `{}` if the attribute doesn't have metadata.
+  Each property in this object represents a metadata element (which name is the name of the property) which value is
+  a JSON object composed of two properties: `value` and `type` (`value` and `type` following the same rules that the
+  ones described in the previous two bullets).
+
+## Direct access to attribute value
+
+Some operations in the NGSIv2 API allow getting/setting directly the attribute value, both in JSON and plain text formats.
+
+In the case of setting value:
+
+* If request payload MIME type is `application/json`, then the value of the attribute is set to the JSON object or array
+  coded in the payload (if the payload is not a valid JSON document, then an error is returned).
+* If request payload MIME type is `text/plain`, then the following algorithm is applied to the payload:
+  * If the payload starts and ends with citation-marks (`"`), the value is taken as a string (citation marks themselves are not
```

Comment:
This is (mostly) a copy of https://github.com/telefonicaid/fiware-orion/issues/1259 item 5, but not sure if is is aligned with our last discussions at skype. Please, @jmcanterafonseca and @kzangeli have a look to it, just in case.